### ISSUE TITLE
sys_refcursor parameter type

### ISSUE THREAD

**Issue by deluvas** [linq2db/linq2db#275]
Hello,

I'm trying to execute a stored procedure from Oracle PL/SQL that's returning results through an OUT SYS_REFCURSOR parameter, however I'm having difficulties making it work since there is no `refcursor` parameter type in linq2db.

Example procedure:

`PROCEDURE my_procedure( p_cr OUT SYS_REFCURSOR )`

How can I achieve this with linq2db?

**Comment by igor-tkachev** [comment]
Can you please provide an example of SQL and C# code?

**Comment by deluvas** [comment]
```SQL
create or replace PROCEDURE MY_PROCEDURE 
(
  crs OUT SYS_REFCURSOR 
) AS 
BEGIN
  OPEN crs FOR
    SELECT * FROM whiskies;
END MY_PROCEDURE;
```

**Comment by rv82** [comment]
Hello!

I interested in this question too. Is it possible to execute Oracle's stored procedures with linq2db and to get a list of records from it through OUT SYS_REFCURSOR parameter, like in previous question?

Thank you!

**Comment by work-des** [comment]
Hello!
How all the same to solve this problem?

**Comment by sdanyliv** [comment]
It is already closed in #555
